.ci/jenkins/Jenkinsfile.post-release

```diff
@@ -116,6 +116,31 @@ pipeline {
                 }
             }
         }
+       stage('update kie-benchmarks main to next SNAPSHOT version') {
+            steps {
+                script {
+                    String prLink = null
+                    String kieBenchmarksRepo = 'kie-benchmarks'
+                    String commitMsg = 'bumped up kie-version to next SNAPSHOT'
+                    String prBranch = 'kie-benchmarks-update'
+                    String targetBranch = 'main'
+                    dir("$kieBenchmarksRepo-bot"){
+                        checkoutRepo(kieBenchmarksRepo, targetBranch)
+                        githubscm.setUserConfigFromCreds()
+                        githubscm.createBranch(prBranch)
+                        // update version of kie-benchmarks
+                        sh "./update-version.sh ${getNextMinorSnapshotVersion(getProjectVersion())}"
```

Comment:
That will rewrite the version for drools as well and we do not want it.
How about changing only version.org.optaplanner property?

Reply:
@dupliaka Hi - I executed locally the update-version script and this changed me 
	modified:   drools-benchmarks/pom.xml
	modified:   optaplanner-benchmarks/optaplanner-perf-benchmark/pom.xml
	modified:   optaplanner-benchmarks/optaplanner-perf-framework/pom.xml
	modified:   optaplanner-benchmarks/pom.xml
	modified:   pom.xml
In no one of these files were changed a version of drools?
Have in mind we are here updating kiegroup/kie-benchmarks. Am I wrong here?

Reply:
Yes that is true, though the benchmarks for drools uses this to define the version of the tested drools: 
 <version.org.drools>${project.version}</version.org.drools>

Reply:
@dupliaka now I see

Reply:
@dupliaka - so there are only three poms to do the change:

Reply:
kie-benchmarks/optaplanner/pom.xml
kie-benchmarks/optaplanner-benchmarks/optaplanner-perf-benchmark/pom.xml
kie-benchmarks/optaplanner-benchmarks/optaplanner-perf-framework/pom.xml
right? 
But kie-benchmarks/optaplanner/pom.xml is pointing to kie-benchmarks/pom.xml - as parent, and this can't be updated since this defines the ${project.version} - so also affects the drools-benchmarks. 
Could you explain t me how you think I should do this?

Reply:
As we discussed with @hellowdan - we need to update versions of OptaPlanner and Drools separately.
So instead of using the script it would be better to change property <version.org.optaplanner> for OptaPlanner release. 
I propose to use maven command 
`mvn versions:set-property -Dproperty=version.org.optaplanner -DnewVersion=_getNextMinorSnapshotVersion_`

Reply:
FYI we have a shared library for that: https://github.com/kiegroup/jenkins-pipeline-shared-libraries/blob/main/vars/maven.groovy#L114

Reply:
Thanks @radtriste just realized that it is there
https://github.com/kiegroup/jenkins-pipeline-shared-libraries/blob/1c85c8815265ff6d37aec4005249299e4ac7d26a/vars/maven.groovy#L118